test/test.hs

```diff
@@ -3944,7 +3948,7 @@ tests = testGroup "hevm"
                 False -> recursiveList ax (a:b)
           recursiveList [] b = pure b
         files <- liftIO $ recursiveList fullpaths []
-        let filesFiltered = filter (\file -> not $ any (`List.isSubsequenceOf` file) ignoredTests) files
+        let filesFiltered = filter (\file -> not $ any (`List.isInfixOf` file) ignoredTests) files
```

Comment:
`isInfixOf` is more strict than `isSubsequenceOf`, right?
But this is comparing paths, no?
Is it going to make any difference here?

Reply:
It's a huge difference :) The `isSubsequenceOf` actually does something _very_ different that what we wanted!

`isSubsequenceOf` will say YES, when "abc" is in the list of exclusions, and the file checked is "sadfasdf-aaaab-bbba-aacc-ccccaa-abbb-bsdf-asdf"

`isInfixOf` will say NO. It will only say YES to: `something-abc-somethingelse` or `abc` or `something-abc` or `abc-something-abc`

This was a bug -- it was meant to be `isInfixOf`. Not `isSubsequenceOf`!